meson.build

```diff
@@ -212,9 +212,9 @@ if enable_logind
   config_h.set10('HAVE_' + func.to_upper(), cc.has_function(func, dependencies: logind_dep))
 
   # systemd unit / service files
+  systemd_dep = dependency('systemd', not_found_message: 'systemd required but not found, please provide a valid systemd user unit dir or disable it')
```

Comment:
This change breaks using elogind (https://bugs.gentoo.org/922458).

Need to keep the session_tracking conditional, I think.

Reply:
I applied this on top which seems to work:
```diff
--- a/meson.build
+++ b/meson.build
@@ -212,11 +212,14 @@ if enable_logind
   config_h.set10('HAVE_' + func.to_upper(), cc.has_function(func, dependencies: logind_dep))
 
   # systemd unit / service files
-  systemd_dep = dependency('systemd', not_found_message: 'systemd required but not found, please provide a valid systemd user unit dir or disable it')
   systemd_systemdsystemunitdir = get_option('systemdsystemunitdir')
-  if systemd_systemdsystemunitdir == '' and session_tracking == 'libsystemd-login'
-    # FIXME: systemd.pc file does not use variables with relative paths, so `define_variable` cannot be used
-    systemd_systemdsystemunitdir = systemd_dep.get_pkgconfig_variable('systemdsystemunitdir')
+  if session_tracking == 'libsystemd-login'
+    systemd_dep = dependency('systemd', not_found_message: 'systemd required but not found, please provide a valid systemd user unit dir or disable it')
+
+    if systemd_systemdsystemunitdir == ''
+      # FIXME: systemd.pc file does not use variables with relative paths, so `define_variable` cannot be used
+      systemd_systemdsystemunitdir = systemd_dep.get_pkgconfig_variable('systemdsystemunitdir')
+    endif
   endif
 
   systemd_sysusers_dir = systemd_dep.get_pkgconfig_variable('sysusers_dir', default: '/usr/lib/sysusers.d')
-- 
2.43.0
```

Reply:
No that suggested change is not right, as the sysuser file is needed independently of logind

Reply:
Sure, but the logic which as-is only installs the sysuser file with systemd anyway.

Reply:
Good point, that needs to be fixed too, as it shouldn't and it needs to always be installed